connection/src/manager.rs

```diff
@@ -40,12 +40,13 @@ impl<C: Connection> ConnectionManager<C> {
                     .into_iter()
                     .map(|conn| {
                         let name = conn.to_string();
-                        let responder_id = conn.uri().responder_id().unwrap_or_else(|_| {
-                            panic!(
-                                "Could not create responder_id from {:?}",
-                                conn.uri().to_string()
-                            )
-                        });
+                        let responder_id =
+                            conn.uri().host_and_port_responder_id().unwrap_or_else(|_| {
+                                panic!(
+                                    "Could not create responder_id from {:?}",
+                                    conn.uri().to_string()
+                                )
+                            });
```

Comment:
Let's include the error 
```suggestion
                            conn.uri().host_and_port_responder_id().unwrap_or_else(|err| {
                                panic!(
                                    "Could not create responder_id from {:?}: {}",
                                    conn.uri().to_string(),
                                    err
                                )
                            });
```

Reply:
Done.